rest_server/src/models/marketplace_item.js

```diff
@@ -62,11 +63,18 @@ class MarketplaceItem {
       if (type && type !== 'all') {
         filterStatement.type = type;
       }
+      if (keyword) {
+        filterStatement[Op.or] = [
+          { name: { [Op.substring]: keyword } },
```

Comment:
what if key word is null

Reply:
keyword=null return same length as query with no "keyword", but may it better to filter "null“ out ?

Reply:
It is ok if there is not error throwed